server-access-nss/src/passwd.rs

```diff
@@ -0,0 +1,69 @@
+use anyhow::Result;
+use libnss::interop::Response;
+use libnss::passwd::{Passwd, PasswdHooks};
+use rusqlite::OpenFlags;
+use rusqlite::{params, Connection, Row, NO_PARAMS};
+
+use crate::db::from_result;
+use crate::db::PATH;
+
+pub struct SqlitePasswd;
+libnss_passwd_hooks!(flantauth, SqlitePasswd);
+
+impl PasswdHooks for SqlitePasswd {
+    fn get_all_entries() -> Response<Vec<Passwd>> {
+        let entries = Connection::open_with_flags(&PATH as &str, OpenFlags::SQLITE_OPEN_READ_ONLY)
+            .map_err(Into::into)
+            .and_then(get_all_entries);
+
+        from_result(entries)
+    }
```

Comment:
Is it need to close the connection?

Reply:
It's embedded into Connection destructor